vpr/src/base/netlist_writer.cpp

```diff
@@ -1170,6 +1143,48 @@ class NetlistWriterVisitor : public NetlistVisitor {
         return io_name;
     }
 
+  protected:
+    /**
+     * @brief Returns the name of a wire connecting a primitive and global net.
+     *
```

Comment:
What does global net mean in this context?  Just a net that is routed?  (I'm guessing that).

Or an actual global (often not routed, on a special network) net?

Would be good to clarify.